.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Thanks for reporting an issue! Please make sure you click the link above to view the issue guidelines, then fill out the blanks below. -->
+
+What are the steps to reproduce this issue?
+-------------------------------------------
+1. …
+2. …
+3. …
+
+What happens?
+-------------
+…
+
+What were you expecting to happen?
+----------------------------------
+…
+
+Any logs, error output, etc?
+----------------------------
+<!-- If it’s long, please paste to https://gist.github.com/ and insert the link here. -->
+
+
+Any other comments?
+-------------------
+…
+
+What versions of software are you using?
+----------------------------------------
+**Operating System:** …
+
+**Toolchain Version:** …
```

Comment:
Let's change this to something like "Python version", "cluster_generator version", "yt version", "unyt version", "other" to make this more explicit.

Reply:
Fixed